yarn.lock

```diff
@@ -2001,6 +2001,14 @@
     normalize-path "^2.0.1"
     through2 "^2.0.3"
 
+"@hapi/basic@^5.1.1":
```

Comment:
I'm not sure, as this is the first time I've ever used Hapi.
I assumed that this would only affect integration tests as it's installed by the Webhook simulator.
Is this not a safe assumption?

In anycase, I've removed it, as I don't actually need authentication, it was just a way of keepign the test code a little cleaner.
I now have a simple "extraction" scheme that gets the credentials out of Hapi when a call is made to the test endpoint.